aten/src/ATen/functorch/BatchRulesReduceOps.cpp

```diff
@@ -367,6 +367,11 @@ std::tuple<Tensor,optional<int64_t>> searchsorted_batch_rule(
   }
   TORCH_INTERNAL_ASSERT(false);
 }
+#define OP_DECOMPOSE(op)  m.impl(#op, static_cast<decltype(&ATEN_FN(op))>(native::op));
+
+TORCH_LIBRARY_IMPL(aten, CompositeImplicitBatched, m) {
+  OP_DECOMPOSE(argsort)
+}
```

Comment:
Let's put OP_DECOMPOSE(argsort) into BatchRulesDecompositions.cpp so we don't need to redefine the macro and add a separate TORCH_LIBRARY_IMPL